tox/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM alpine:3
 
 ENV PATH="$PATH:/root/.pyenv/bin:/root/.pyenv/shims"
-ENV version=3.21.2
+ENV version=3.21.3
 
-RUN apk add --no-cache --virtual=.build-deps curl git linux-headers openssl-dev sqlite-dev readline-dev bzip2-dev ncurses-dev sqlite-dev patch xz-dev zlib-dev && \
+RUN apk add --no-cache --virtual=.build-deps curl git linux-headers openssl-dev sqlite-dev readline-dev bzip2-dev ncurses-dev sqlite-dev patch xz-dev zlib-dev cargo && \
```

Comment:
nitpick: but can you add it so it's sorted alphabetically like the other packages?

Reply:
ok, will order also others than :D i checked it and it was not all in alphabetical order so i just added it at the end...but sure, will change

Reply:
damn... 😅️
sorry then
but better now